app/main.py

```diff
@@ -1,8 +1,39 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
```

Comment:
Place each argument in the `__init__` on a new line in the `Car` and `CarWashStation` classes, so that there would be no arguments on the same line with the opening and closing parenthesis to improve the code readability.